### ISSUE TITLE
Missing file "papilolib_export.h"?

### ISSUE THREAD

**Issue by s-fleuren** [scipopt/papilo#44]
Hi, I'm trying to use the C API found in src/papilolib.h, but this includes "papilolib_export.h" which I can't find anywhere. Without it PAPILOLIB_EXPORT is not defined. 

I've build papilo using the basic build instructions
mkdir build
cd build
cmake ..
make

Is using the API in src/papilolib.h supported, and if yes how do I obtain/create papilolib_export.h?

**Comment by alexhoen** [comment]
Hi,

If I am understanding you correctly, you want to integrate PaPILO into your C Project via the API. Unfortunately, the repository is not really cleaned up. We need to do this. Nevertheless it is possible to integrate it into your project.
Assuming you are using CMake, then I would recommend taking a look at the integration of PaPILO in SCIP or SoPlex .

You would then need something like 

`c
option(PAPILO "should papilo library be linked" OFF)
# look for presolvelib PaPILO
if(PAPILO)
    set(PAPILO_MIN_VERSION 2.0.0.0)
    message(STATUS "Finding PAPILO")
    find_package(PAPILO CONFIG ${PAPILO_MIN_VERSION} REQUIRED HINTS ${PAPILO_DIR})
    set(libs ${libs} ${PAPILO_IMPORTED_TARGETS})
    message(STATUS "Finding PAPILO - found")
    set(SOPLEX_WITH_PAPILO on)
        find_package(Boost 1.65.0) # PaPILO requires at least Boost 1.65 (on mac 1.72)

else()
    message(STATUS "Support PAPILO: OFF")
    set(PAPILO_IMPORTED_TARGETS "")
    set(SOPLEX_WITH_PAPILO off)
endif()
`

_basically you can search in the soplex repository for "papilo". The results should be limited._

Here is the CMake file of Soplex:
https://github.com/scipopt/soplex/blob/master/CMakeLists.txt

And here how you can use it:
https://github.com/scipopt/soplex/blob/master/src/soplex/spxpapilo.h

If you have further question don't hesitate to ask.

**Comment by s-fleuren** [comment]
Thank you for the quick response. I was hoping to be able to use the src/papilolib.h since it looks much more user friendly than the C api soplex provides. But if this is not supported yet I'll look into some alternative libraries first. Feel free to close the issue if you wish.

**Comment by alexhoen** [comment]
>   since it looks much more user friendly than the C api soplex provides.

This is not about using the Soplex interface rather than using the integration of PaPILO into Soplex as an example.
